index.html

```diff
@@ -250,7 +249,44 @@ <h3 class="title">Project Tile goes here</h3>
                 </div>
             </div>
         </section>
-    </div>
+
+        <footer class="information-footer">
+
+            <div class="contacts-foolter">
+                <div class="logo-footer">
+                    <a href="index.html"><img src="images/logo 1 (2).svg" alt="logo"></a>
+                </div>
+
+                <div class="content-foolter">
+                    <p class="text-footer">+55 83 988970601</p>
+                    <p class="text-footer">mikael.mesg@gmail.com</p>
+                    <a class="icon-footer" href=""><img src="images/Vector (2).svg" alt="logo-gthub"></a>
+                    <a class="icon-footer" href=""><img src="images/ant-design_twitter-circle-filled (1).svg" alt="logo-twitter"></a>
+                    <a class="icon-footer" href=""><img src="images/entypo-social_linkedin-with-circle (1).svg" alt="logo-linkedin"></a>
+                </div>
+            </div>
+
+            <div class="division">
+                <hr>
+                </hr>
+            </div>
+
+            <div class="navigation-footer">
+                <div class="navigation-end">
+                    <a class="link-footer" href="">Home</a>
+                    <a class="link-footer" href="">About</a>
+                    <a class="link-footer" href="">Tech Attack</a>
+                    <a class="link-footer" href="">Projects</a>
+                    <a class="link-footer" href="">Contact</a>
+                </div>
+
+                <div class="credit">
+                    <p>Designed and built by <span class="highlight-footer">Mikael Elias</span></p>
```

Comment:
Já que a gente não tá pagando por esse layout acho que a gente pode pelo menos dar credito pra o designer, então eu incluiria o nome dele nos créditos, depois a gente pode achar o link do perfil dele pra colocar tb.
```suggestion
                    <p>Designed by <span class="highlight-footer">Pavan MG</span> and built by <span class="highlight-footer">Mikael Elias</span></p>
```

Reply:
feito!